056-CosmicTroubleshooting/Student/Challenge-05.md

```diff
@@ -0,0 +1,21 @@
+# Challenge 05 - Going Global
+
+[< Previous Challenge](./Challenge-04.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-06.md)
+
+## Introduction
+
+Stuff and More Stuff Co. has is expanding their operations across the globe. Since their user base is growing at a rapid pace in Europe, they would like to have their database and application available across both the US and Europe.
```

Comment:
Remove "has" at the beginning of the first sentence.

Reply:
Thank you Pete! Changed as per above.